src/main/resources/currentWeather.yaml

```diff
@@ -0,0 +1,60 @@
+swagger: '2.0'
+info:
+  description: Showing current weather
+  version: '0.1'
+  title: Current weather
+tags:
+  - name: superhero
```

Comment:
uhm.... copy paste from somewhere? :D 

But yeah, this tag doesn't look correct, it basically defines what the name of the controller / API interface would be that it generates.